scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -773,6 +772,44 @@ TDFN-6-1EP_2.5x2.5mm_P0.65mm_EP1.3x2mm:
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
 
+TDFN-14-1EP_3x3mm_P0.4mm_EP1.78x2.35mm:
+  device_type: 'TDFN'
+  size_source: 'https://pdfserv.maximintegrated.com/land_patterns/90-0063.PDF (T1433-2C)'
+  body_size_x: 3 +/-0.1
+  body_size_y: 3 +/-0.1
```

Comment:
the datsheet uses min/nom/max so use the same here (we don't trust ourself to do calculations for good reason)